src/lib/stable/execute_with_candid_serde.ts

```diff
@@ -12,7 +12,8 @@ export type CanisterMethodMode =
     | 'postUpgrade' // After canister upgrade
     | 'preUpgrade' // Before canister upgrade
     | 'inspectMessage' // Message inspection
-    | 'heartbeat'; // Periodic heartbeat
+    | 'heartbeat' // Periodic heartbeat
+    | 'onLowWasmMemory'; // Low WASM memory handler
```

Comment:
We should write `Wasm` instead of `WASM` everywhere in comments.

Reply:
done

Reply:
Actually you introduced a comment with `wasm`. Can you do a search through the whole codebase and update all case insensitive uses of `wasm` in comments to `Wasm`?

Reply:
done